.github/workflows/openjdk_14_build.yml

```diff
@@ -0,0 +1,117 @@
+name: openjdk_14_build
+on:
+  pull_request:
+  push:
+    branches: [ main, develop ]
+    tags: [ v* ]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: gcr.io/swirlds-registry/cci-openjdk-ghr:14.0.2
+    env:
+      MAVEN_OPTS: -Xmx5200m
+      LC_ALL: C.UTF-8
+      DEBIAN_FRONTEND: noninteractive
+    timeout-minutes: 60
+    steps:
+      - run: apt-get -y install software-properties-common
+      - run: add-apt-repository ppa:git-core/ppa
+      - run: apt-get -y install git
```

Comment:
I don't think git is necessary to install as it should be there by default for use by the checkout action. What is `software-properties-common` for and is it necessary?

Reply:
the gcr.io/swirlds-registry/cci-openjdk-ghr:14.0.2 container has an older version of git that doesn't work with the git checkout action.